package.json

```diff
@@ -62,6 +62,7 @@
 		"onCommand:cosmosDB.deleteDocDBDocument",
 		"onCommand:cosmosDB.deleteGraphDatabase",
```

Comment:
I got this message:

![image](https://user-images.githubusercontent.com/6913354/46632959-2ae72f80-cb01-11e8-9d16-bb97e27b33a4.png)

Reply:
I seem to always get this when imported into mongodb.